src/ukify/ukify.py

```diff
@@ -988,7 +992,13 @@ def make_uki(opts):
     # UKI creation
 
     if linux is not None:
-        uki.add_section(Section.create('.linux', linux, measure=True))
+        try:
+            virtual_size = pefile.PE(linux, fast_load=True).OPTIONAL_HEADER.SizeOfImage
+        except pefile.PEFormatError:
+            print(f"{f} is not a valid PE file, not using SizeOfImage.")
```

Comment:
shouldn't we fail fatally in this case btw?

Reply:
No because this used to work, we had test failures when it was fatal, so it'd be a backwards compat break if we suddenly started insisting on .linux being a PE